lib/atoms/label/example.html

```diff
@@ -0,0 +1 @@
+<label for=" " /></label>
```

Comment:
We need to add an angular parameter for "for" attribute (e.g. for="{{inputId}}"). Also class attribute must be added, in order to customize the example from patterns configuration.